src/main/java/com/redhat/jenkins/plugins/ci/CIBuildTrigger.java

```diff
@@ -173,7 +178,7 @@ private void startTriggerThread() {
 			}
 		}
 		if (job instanceof AbstractProject) {
```

Comment:
Do we need this if block anymore? I think workflow jobs support disable now..

Reply:
I have no idea.  I didn't change it because I wouldn't know.

Reply:
Scott, can we get this PR merged? If workflow jobs support disable now I think that is a different task.....

Reply:
I agree this can be in a different PR